test/test_direncryption.py

```diff
@@ -122,13 +126,14 @@ def test_encrypt(expanduser, GPGOps):
     expanduser.side_effect = [
         saved_params['plaindir'],
         saved_params['securedir'],
+        saved_params['restoredir'],
         saved_params['gpg_homedir'],
         saved_params['gpg_binary']
     ]
     inventory = MagicMock()
 
     de = DirEncryption(test_args)
-    de.encrypt('plainfile', 'securefile', inventory)
+    de.encrypt('plainfile', 'securefile', inventory, 0)
```

Comment:
Instead of `0`, better `False` or `is_link=False`.
```suggestion
    de.encrypt('plainfile', 'securefile', inventory, `is_link=False`)
```

Reply:
Removed `is_link` parameter because default value is now `False`